realm/realm-library/src/main/cpp/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ set(classes_LIST
     io.realm.internal.Table io.realm.internal.CheckedRow
     io.realm.internal.LinkView io.realm.internal.Util io.realm.internal.UncheckedRow
     io.realm.internal.TableQuery io.realm.internal.SharedRealm io.realm.internal.TestUtil
-    io.realm.log.LogLevel io.realm.log.RealmLog io.realm.Property io.realm.RealmSchema
+    io.realm.log.LogLevel io.realm.log.RealmLog io.realm.Property io.realm.RealmNativeSchema
```

Comment:
Perhaps it was a Slack discussion, but #4215 captures the conclusion. I think it sprang from the fine-grained single object notifications that needed a wrapper around Object Stores `Object` class, which obviously doesn't work. `NativeObject` was already taken by our phantom reference interface so prefixing all wrappers around Object Store classes with `Os*` seemed the next best thing.

#4215 Just captured the conclusion.